execution/executor/src/lib.rs

```diff
@@ -726,17 +726,19 @@ impl<V: VMExecutor> BlockExecutor for Executor<V> {
         let version = ledger_info_with_sigs.ledger_info().version();
         let num_persistent_txns = self.cache.synced_trees().txn_accumulator().num_leaves();
 
-        if version + 1 < num_persistent_txns {
-            return Err(Error::InternalError {
-                error: format!(
-                    "Try to commit stale transactions with the last version as {}",
-                    version
-                ),
-            });
-        }
+        if num_persistent_txns > 0 {
```

Comment:
Why the change in logic? Old bug?

Reply:
fuzzer reports a case that the addition in old code can overflow.